includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -506,6 +525,7 @@ private function getVerticalData() {
 		 * - global-footer-licensing-and-vertical-description-param-vertical-music
 		 * - global-footer-licensing-and-vertical-description-param-vertical-comics
 		 * - global-footer-licensing-and-vertical-description-param-vertical-movies
+		 * - global-footer-licensing-and-vertical-description-param-vertical-fandom // TODO: add to message files
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)